packages/frontend/web-ui/src/auth/hooks/useLoginForm.spec.ts

```diff
@@ -0,0 +1,313 @@
+import { describe, expect, jest, it, beforeAll, afterAll } from '@jest/globals';
+
+jest.mock('../../common/helpers/validateEmail');
+jest.mock('../../common/helpers/validatePassword');
+jest.mock('../helpers/isFullfilledPayloadAction');
+jest.mock('react-redux', () => ({
+  ...(jest.requireActual('react-redux') as Record<string, unknown>),
+  useDispatch: jest.fn(),
```

Comment:
We are not importing this, mock `useAppDispatch` instead.

Reply:
Done!

Reply:
We're still mocking this. As said, let's mock `useAppDispatch` instead.

Reply:
Done!